src/test/java/seedu/address/storage/JsonAdaptedPersonTest.java

```diff
@@ -103,8 +114,18 @@ public void toModelType_invalidTags_throwsIllegalValueException() {
         List<JsonAdaptedTag> invalidTags = new ArrayList<>(VALID_TAGS);
         invalidTags.add(new JsonAdaptedTag(INVALID_TAG));
         JsonAdaptedPerson person =
-                new JsonAdaptedPerson(VALID_NAME, VALID_PHONE, VALID_EMAIL, VALID_ADDRESS, invalidTags);
+                new JsonAdaptedPerson(VALID_NAME, VALID_PHONE, VALID_EMAIL, VALID_ADDRESS, invalidTags,
+                        VALID_POLICIES);
         assertThrows(IllegalValueException.class, person::toModelType);
     }
 
+    @Test
```

Comment:
Nice implementation of testing invalid `premium` value when creating `Policy`! Perhaps can extend to test invalid `name`, `company`, `policyManager` in the future 😀